sdk/communication/azure-communication-phonenumbers/CHANGELOG.md

```diff
@@ -16,6 +16,9 @@
 - Users can now manage SIP configuration for Direct routing.
 - Adds support for Azure Communication Services Phone Numbers Browse API Methods.
 
+### Bug Fixes
```

Comment:
I would argue this is not necessary in the changelog. It is supposed to contain end-user relevant changes, which this is really not since it is only a change in our unit tests.